test/client/fixtures/sandbox/iframe-flag-test.js

```diff
@@ -0,0 +1,299 @@
+var urlUtils = hammerhead.get('./utils/url');
+
+var iframeSandbox = hammerhead.sandbox.iframe;
+var nativeMethods = hammerhead.nativeMethods;
+var browserUtils  = hammerhead.utils.browser;
+
+QUnit.testStart(function () {
+    // NOTE: The 'window.open' method used in QUnit.
+    window.open       = nativeMethods.windowOpen;
+    window.setTimeout = nativeMethods.setTimeout;
+    iframeSandbox.on(iframeSandbox.RUN_TASK_SCRIPT, initIframeTestHandler);
+    iframeSandbox.off(iframeSandbox.RUN_TASK_SCRIPT, iframeSandbox.iframeReadyToInitHandler);
+});
+
+QUnit.testDone(function () {
+    iframeSandbox.off(iframeSandbox.RUN_TASK_SCRIPT, initIframeTestHandler);
+});
+
+function hasIframeFlag (url) {
+    var resourceType = urlUtils.parseProxyUrl(url).resourceType;
+
+    return !!resourceType && resourceType.indexOf('i') !== -1;
+}
+
+function provokeTargetCalculation (link) {
+    link.click();
+}
+
+var tagUrlAttr = {
+    a:    'href',
+    form: 'action',
+    area: 'href',
+    base: 'href'
+};
+
+module('target');
+
+test('assign a url attribute to elements with the "target" attribute in top window', function () {
```

Comment:
Move all `target` specific tests to the https://github.com/DevExpress/testcafe-hammerhead/blob/master/test/client/fixtures/target-attr-test.js